packages/core/entity/FileDetail/index.ts

```diff
@@ -193,8 +196,13 @@ export default class FileDetail {
             return undefined;
         }
 
-        const baseURLHttp = baseURL.replace("https", "http");
-        return `${baseURLHttp}/labkey/aics_microscopy/AICS/editPlate.view?Barcode=${platebarcode}`;
+        let labkeyHost = "localhost:9081";
+        if (baseURL === FileExplorerServiceBaseUrl.PRODUCTION) {
+            labkeyHost = "aics.corp.alleninstitute.org";
+        } else if (baseURL === FileExplorerServiceBaseUrl.STAGING) {
+            labkeyHost = "stg-aics.corp.alleninstitute.org";
+        }
+        return `http://${labkeyHost}/labkey/aics_microscopy/AICS/editPlate.view?Barcode=${platebarcode}`;
```

Comment:
I'm fairly certain this is a labkey issue and not us, but just in case it's a link thing: noticing that for the ones I tested with this I kept getting a 403 error on the labkey page (no permission to view content)